packages/components/src/mobile/alt-text-settings/styles.scss

```diff
@@ -0,0 +1,29 @@
+.altTextEditor {
+	padding: 14px 0;
+	min-height: 80px;
+	max-height: 250px;
+}
+
+.altTextEditorDark {
+	color: $white;
+}
+
+.altTextFootnote {
+	font-size: $default-font-size;
+	color: $gray-text-min;
+	padding: 8px 16px 24px;
+}
+
+.altTextLink {
+	color: $blue-wordpress;
+}
+
+.horizontalBorder {
+	border-top-width: $border-width;
+	border-bottom-width: $border-width;
+	border-color: $light-gray-400;
```

Comment:
I don't think these border colors are the right ones, but it doesn't look like we have the ones in there that I'd have assumed we could use. 🤔 cc @Guarani Did we ever finish moving all of the border-colors over to the "new" neutral colors? I'm not seeing them in `colors.native.scss` but I was almost positive that we added some color variables for border/separator during last hack week. Maybe they're in a different file?

Reply:
For reference, I got these colours from looking at the following other instances of separators within the app:

* https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/mobile/bottom-sheet/styles.native.scss#L19
* https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/mobile/bottom-sheet/styles.native.scss#L89
* https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/mobile/color-settings/style.native.scss#L2
* https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/mobile/picker/styles.native.scss#L1

Let me know if there's another way to gather the correct ones!

Reply:
We didn't finish @iamthomasbishop 😞 I remember still having some confusion about the changes we were supposed to make. There's a PR open [here](https://github.com/WordPress/gutenberg/pull/25429) and I think the internal discussion was still on-going when we paused this after that HACK week finished.

Reply:
The PR I think you're remembering is [this one](https://github.com/WordPress/gutenberg/pull/25350/files). I do think we need regroup and continue this. I remember some confusion about how to map the old colors to the new, so maybe we could start again by creating an issue that specifies the work needed to be done and we go from there, wdyt?
Sorry to discuss a separate issue on this PR.

Reply:
> For reference, I got these colours from looking at the following other instances of separators within the app:

Ahh, fair enough! I knew we hadn't updated the grays for the separators yet, but I wasn't sure which color was being used. I'd like to go through at some point and make sure all of our separators/borders are using the same "new" color (which as @Guarani pointed out looks like it didn't quite get shipped) but let's leave that for another time. In other words, this is fine for now because it's consistent w/ the other border colors 😄